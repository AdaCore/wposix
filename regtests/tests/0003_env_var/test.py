from test_support import *

gprbuild('demo5')
run('demo5')
