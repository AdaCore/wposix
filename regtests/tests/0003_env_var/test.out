HOME = <home>
MYENV = val
OS = Windows_NT
PATH = <path>
PWD = <pwd>
USER = <user>
1 ----------
HOME = <home>
MYENV = val
OS = Windows_NT
PATH = <path>
PWD = <pwd>
USER = <user>
2 ----------
HOME = <home>
OS = Windows_NT
PATH = <path>
PWD = <pwd>
USER = <user>
ada = this is ada
3 ----------
4 ----------
5 ----------
length  1
Only = one
